opni-insights-service/app/main.py

```diff
@@ -622,95 +741,31 @@ async def index_logs(start_ts: int, end_ts: int, scroll_id: Optional[str] = None
         # Bad Request
         logging.error(e)
 
-
 @app.get("/areas_of_interest")
-async def get_areas_of_interest(start_ts: int, end_ts: int):
+async def index_areas_of_interest(start_ts: int, end_ts: int):
```

Comment:
what's the benefit of having another function `index_areas_of_interest()`, which does nothing else than simply calls `get_areas_of_interest()`?

Reply:
It's to follow the consistency of all the other endpoints in the insights service.